applications/openshift/logging/audit_profile_set/rule.yml

```diff
@@ -58,7 +58,7 @@ references:
   nerc-cip: CIP-003-8 R4,CIP-003-8 R4.1,CIP-003-8 R4.2,CIP-003-8 R5.2,CIP-003-8 R6,CIP-004-6 R2.2.2,CIP-004-6 R2.2.3,CIP-004-6 R3.3,CIP-007-3 R.1.3,CIP-007-3 R5,CIP-007-3 R5.1.1,CIP-007-3 R5.2,CIP-007-3 R5.3.1,CIP-007-3 R5.3.2,CIP-007-3 R5.3.3,CIP-007-3 R6.5
   nist: AU-2,AU-3,AU-3(1),AU-6,AU-6(1),AU-7,AU-7(1),AU-8,AU-8(1),AU-9,AU-12,AU-12(1),AU-12(3),CM-5(1),SI-11,SI-12,SI-4(20),SI-4(23)
   pcidss: Req-2.2,Req-12.5.5
-  srg: SRG-APP-000089-CTR-000150,SRG-APP-000090-CTR-000155,SRG-APP-000091-CTR-000160,SRG-APP-000095-CTR-000170,SRG-APP-000096-CTR-000175,SRG-APP-000097-CTR-000180,SRG-APP-000098-CTR-000185,SRG-APP-000099-CTR-000190,SRG-APP-000100-CTR-000195,SRG-APP-000100-CTR-000200,SRG-APP-000101-CTR-000205,SRG-APP-000116-CTR-000235,SRG-APP-000118-CTR-000240,SRG-APP-000119-CTR-000245,SRG-APP-000120-CTR-000250,SRG-APP-000121-CTR-000255,SRG-APP-000122-CTR-000260,SRG-APP-000123-CTR-000265,SRG-APP-000181-CTR-000485,SRG-APP-000266-CTR-000625,SRG-APP-000374-CTR-000865,SRG-APP-000375-CTR-000870,SRG-APP-000380-CTR-000900,SRG-APP-000381-CTR-000905,SRG-APP-000492-CTR-001220,SRG-APP-000493-CTR-001225,SRG-APP-000494-CTR-001230,SRG-APP-000495-CTR-001235,SRG-APP-000496-CTR-001240,SRG-APP-000497-CTR-001245,SRG-APP-000498-CTR-001250,SRG-APP-000499-CTR-001255,SRG-APP-000500-CTR-001260,SRG-APP-000501-CTR-001265,SRG-APP-000502-CTR-001270,SRG-APP-000504-CTR-001280,SRG-APP-000507-CTR-001295,SRG-APP-000508-CTR-001300,SRG-APP-000509-CTR-001305,SRG-APP-000510-CTR-001310
```

Comment:
Looking through a few that are being removed and they're just here because they have something to do with logging?

We're replacing this list with just the SRGs that are specific to audit logging?

Reply:
Yes, the original SRG IDs were mass-added in 750c645772449b8522557ec1eea36e4044bc5a14 based on mapping between NIST controls and SRG-APP IDs. But the mass-add did not distinguish between "related" rules and "fixed-by" rules, while the SRG export script puts the fix and check of any rule that has the SRG ID in its rule.yaml to the export.

btw also RHEL-8/9 STIG is coded up the same way. We can still use `related_rules` in the control files to give some additional context, but we won't be including the rule's text in the export.

Reply:
But you are right I should clean up the other rules, too. I'll do it now, thank you!

Reply:
PR extended. There's a couple more patches now that address all the controls that I removed from the SRG list in this rule. Those controls that could be addressed either by saying that the control is inherently met, does not apply or does not meet have status justifications now. There's three controls that are covered by new manual rules. And finally the last patch is a grab-bag of controls where I just removed this rule from the `rules:` list, but those controls will be addressed by linking RHEL/RHCOS controls later. Because we need to do adjustments to the build system, I can't do it now, for now, let's just make sure we don't cover the controls by a rule that's not really related.